src/current/v23.2/configure-logs.md

```diff
@@ -257,9 +257,10 @@ Along with the [common sink parameters](#common-sink-parameters), each HTTP serv
 | `disable-keep-alives` | When `true`, disallows reuse of the server connection across requests.<br><br>**Default:** `false` (reuses connections)                                                                                                                                                                                                                                            |
 | `compression`         | Compression method for the HTTP request body. Valid values `gzip` or `none`.<br><br>**Default:** `gzip` |
 | `headers`             | Map of key-value string pairs which will be appended to every request as custom HTTP headers. |
+|<a id="file-based-headers"></a> `file-based-headers`  | Map of key-filepath string pairs. The file specified by the filepath only contains a value that corresponds to the key. The key from the YAML configuration and the value contained in the file will be appended to every request as a custom HTTP header.<br><br>For example:  `file-based-headers: {X-CRDB-HEADER-KEY: /some/path/to/file_with_value, X-ANOTHER-HEADER-KEY: /other/path/to/file_with_another_value}`<br><br>A value in a file can be updated without restarting the `cockroach` process. Send SIGHUP to the `cockroach` process to notify it to refresh values.|
```

Comment:
I'm guessing these `{` and `}` placeholders are here based on [this guidance from our style guide](https://github.com/cockroachdb/docs/blob/main/StyleGuide.md#placeholders).  However in this instance our style guide is wrong IMO, and these brackets made things more confusing. I only understood what they mean because of our style guide, but I would not understand them if I were an outside user reading these docs

I recommend removing the brackets so this line reads:

```
file-based-headers: X-CRDB-HEADER-KEY: /some/path/to/file_with_value, X-ANOTHER-HEADER-KEY: /other/path/to/file_with_another_value
```

Reply:
In YAML, curly brackets {} are used to define mappings (associative arrays or dictionaries). Mappings in YAML represent key-value pairs, where each key is associated with a corresponding value. The curly brackets are used to encapsulate the key-value pairs within a mapping.

So I need to leave the curly brackets in this example.

Reply:
thanks for explaining that, for some reason (probably not reading carefully) I didn't realize this was happening inside YAML!